.github/workflows/ci-events-helsinki-staging.yml

```diff
@@ -0,0 +1,129 @@
+  NEXT_PUBLIC_SENTRY_ENVIRONMENT: 'test'
+  NEXT_PUBLIC_SENTRY_DSN: ${{ secrets.GH_SENTRY_DSN }}
+  NEXT_PUBLIC_SENTRY_TRACE_SAMPLE_RATE: 0
+  NEXTJS_DISABLE_SENTRY: 1
+  SENTRY_AUTH_TOKEN: ${{ secrets.GH_SENTRY_AUTH_TOKEN }}
+  NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT: https://tapahtumat.hkih.stage.geniem.io/graphql
+  NEXT_PUBLIC_EVENTS_GRAPHQL_ENDPOINT: https://tapahtumat-proxy.test.kuva.hel.ninja/proxy/graphql
+  NEXT_PUBLIC_LINKEDEVENTS_EVENT_ENDPOINT: https://api.hel.fi/linkedevents/v1/event
+  NEXT_PUBLIC_MATOMO_URL_BASE: '//webanalytics.digiaiiris.com/js/'
+  NEXT_PUBLIC_MATOMO_SITE_ID: 939
+  NEXT_PUBLIC_MATOMO_SRC_URL: 'piwik.min.js'
+  NEXT_PUBLIC_MATOMO_TRACKER_URL: 'tracker.php'
+  NEXT_PUBLIC_MATOMO_ENABLED: 0
+  NEXT_PUBLIC_APP_ORIGIN: https://${{ secrets.ENVIRONMENT_URL_STAGING }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        uses: andersinno/kolga-build-action@v2
+        env:
+          DOCKER_IMAGE_NAME: events-helsinki-staging
+          DOCKER_BUILD_ARG_APP_PORT: ${{ env.SERVICE_PORT }}
+          DOCKER_BUILD_ARG_PROJECT: ${{ env.PROJECT }}
+          DOCKER_BUILD_ARG_SENTRY_AUTH_TOKEN: ${{ secrets.GH_SENTRY_AUTH_TOKEN }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_DEBUG: 1
+          DOCKER_BUILD_ARG_NEXTJS_DISABLE_SENTRY: ${{env.NEXTJS_DISABLE_SENTRY}}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT: ${{ env.NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_EVENTS_GRAPHQL_ENDPOINT: ${{ env.NEXT_PUBLIC_EVENTS_GRAPHQL_ENDPOINT }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_LINKEDEVENTS_EVENT_ENDPOINT: ${{ env.NEXT_PUBLIC_LINKEDEVENTS_EVENT_ENDPOINT }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_APP_ORIGIN: ${{ env.NEXT_PUBLIC_APP_ORIGIN }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_URL_BASE: ${{ env.NEXT_PUBLIC_MATOMO_URL_BASE }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_SITE_ID: ${{ env.NEXT_PUBLIC_MATOMO_SITE_ID }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_SRC_URL: ${{ env.NEXT_PUBLIC_MATOMO_SRC_URL }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_TRACKER_URL: ${{ env.NEXT_PUBLIC_MATOMO_TRACKER_URL }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_MATOMO_ENABLED: ${{ env.NEXT_PUBLIC_MATOMO_ENABLED }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_ALLOW_UNAUTHORIZED_REQUESTS: 1
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_SENTRY_ENVIRONMENT: 'staging'
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_SENTRY_DSN: ${{ secrets.GH_SENTRY_DSN }}
+          DOCKER_BUILD_ARG_NEXT_PUBLIC_SENTRY_TRACE_SAMPLE_RATE: ${{ env.NEXT_PUBLIC_SENTRY_TRACE_SAMPLE_RATE }}
+  staging:
+    runs-on: ubuntu-latest
+    needs: build
+    name: Staging and Accept
+    steps:
+      - uses: actions/checkout@v2
+      - uses: andersinno/kolga-setup-action@v2
+
+      - name: Deploy
+        uses: andersinno/kolga-deploy-action@v2
+        with:
+          track: 'staging'
+        env:
+          K8S_NAMESPACE: ${{ secrets.K8S_NAMESPACE_STAGING }}
```

Comment:
Both services needs own namespace, this needs to variate too

Reply:
added `K8S_NAMESPACE_STAGING_HARRASTUKSET` = `events-helsinki-monorepo-hobbies-qa` and `K8S_NAMESPACE_STAGING_TAPAHTUMAT` = `events-helsinki-monorepo-events-qa`